opensearch/src/main/java/org/opensearch/sql/opensearch/setting/OpenSearchSettings.java

```diff
@@ -229,6 +229,13 @@ public class OpenSearchSettings extends Settings {
           Setting.Property.NodeScope,
           Setting.Property.Dynamic);
 
+  public static final Setting<?> FIELD_TYPE_TOLERANCE_SETTING =
+      Setting.boolSetting(
+          Key.FIELD_TYPE_TOLERANCE.getKeyValue(),
+          false,
```

Comment:
Set default = true. the reasons is most of user can get correct result ( similar to DSL response ). Even it is breaking current default behavior.

Reply:
OK.

Reply:
Default is now true.